model/client.yaml

```diff
@@ -12,27 +12,35 @@ groups:
     attributes:
       - id: address
         type: string
-        brief: Client address - unix domain socket name, IPv4 or IPv6 address.
+        brief: Client address - IP address or Unix domain socket name.
```

Comment:
Either we need allow a domain name in the client.address as well, or we forbid it in the server address.

Reply:
is there a case when we know a client domain name? 
The asymmetry here is caused by technical asymmetry - clients don't have domain names.

At the same time changing this description and allowing domain name (if there is a tech that needs it) would not be a breaking change.

Reply:
(and if we wanted to set client domain name on a client span, we'd probably prefer  to do it on `host.name` resource attribute rather than `client.address`)

Reply:
@Oberon00 since this PR isn't changing the status quo, are you ok with merging as-is and we can keep iterating on trying to address all of your concerns expressed in #206?

Reply:
@lmolkova Host resource attribute is a good point. While a host can have multiple addresses & domain names and there could be a paricular one that is relevant for a particular operation, it definitely reduces the importance. Still, the difference should be called out & justified in the description.

@trask Fair enough

Reply:
created #251